src/net/sourceforge/kolmafia/KoLCharacter.java

```diff
@@ -1,5 +1,7 @@
 package net.sourceforge.kolmafia;
 
+import static net.sourceforge.kolmafia.Modifiers.MUS_EXPERIENCE_PCT;
```

Comment:
Might be better to either import everything statically or nothing? Just one looks a bit odd.

Reply:
Mistake, thanks.